ui/lib/core/addon/components/replication-header.js

```diff
@@ -22,6 +22,7 @@ import layout from '../templates/components/replication-header';
 
 export default Component.extend({
   layout,
+  data: null,
```

Comment:
Is this change part of the test fixes, or why is it necessary?

Reply:
ah yes, this was to ensure the property was documented & would show up in the Ember debugger. previously `data` was being used in the template but wasn't explicitly referenced in the component JS file, which meant that the component in the Ember debugger didn't have a `data` attribute. i needed access to it to debug the tests.

you can see that the attribute is now listed here:
![image](https://user-images.githubusercontent.com/903288/83695039-2d3bae80-a5ae-11ea-90a2-7250ded349d4.png)

so fun fact: defining a default value (even if it's `null`) in a component ensures it's listed in the Ember debugger too!